R/tm_g_patient_profile.R

```diff
@@ -562,94 +578,32 @@ srv_g_patient_profile <- function(id,
       # get ADSL dataset ---
       ADSL <- data[[sl_dataname]]() # nolint
 
-      if (!is.null(input$select_ex)) {
-        if (input$select_ex == FALSE | is.na(ex_dataname)) {
-          ADEX <- NULL # nolint
-        } else {
-          ADEX <- data[[ex_dataname]]() # nolint
-          validate_has_variable(ADEX, adex_vars)
-        }
-      } else {
-        ADEX <- NULL # nolint
+      ADEX <- NULL # nolint
+      if ((ex_dataname %in% input$select_ADaM) && !is.na(ex_dataname)) {
```

Comment:
if `ex_dataname` will be empty then `select_plot()[ex_dataname]` will never be true 
```suggestion
      if (isTRUE(select_plot()[ex_dataname])) {
```